src/interface/metadata.tsx

```diff
@@ -115,7 +115,7 @@ export const defaultPeep: Trait[] = [
 	},
 	{
 		trait_type: 'Outerwear',
-		value: 'None',
+		value: 'Tan Hoodie',
```

Comment:
Will having a hoodie in this scene cause too many exclusions in the wardrobe scene on load?

Reply:
It definitely causes some, but more than that.. if you are a sane human you start from the beginning with 'Tops' and none of them are visible under the hoodie lol.

However I know @misskatles was really keen on having the hoodie for this scene. Maybe we could update the metadata on wardrobe load?

Reply:
> if you are a sane human you start from the beginning
@misskatles is def not a sane human then 😂 She was all over the place

A switch from hoodie on wardrobe load is another point of failure. I'd like to avoid if @misskatles is ok with the tank in this scene instead